fdbserver/DataDistribution.actor.cpp

```diff
@@ -81,6 +81,11 @@ ShardSizeBounds ShardSizeBounds::shardSizeBoundsBeforeTrack() {
 		                                                      .opsReadPerKSecond = StorageMetrics::infinity } };
 }
 
+enum class DDAuditContext : uint8_t {
```

Comment:
Do we still need `DDAuditContext`? It does not take effects any more.

Reply:
Yes. Removed